lib/galaxy/webapps/galaxy/config_schema.yml

```
@@ -53,7 +53,6 @@ mapping:
           config file documentation.
 
           Sample default 'sqlite:///./database/universe.sqlite?isolation_level=IMMEDIATE'
```

I think you missed this one:

```suggestion
          Sample default 'sqlite:///<data_dir>/universe.sqlite?isolation_level=IMMEDIATE'
```

---

Fixing it now.